Python/Source/Sample.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Sample.hpp"
+
+using namespace pybind11;
+
+namespace Rover {
+  void export_sample(module& module) {
+    class_<PythonSample>(module, "Sample")
+      .def(init<>())
```

Comment:
Provide a constructor that initializes the values directly.

Reply:
The C++ Sample does not have such a constructor. Should I create one on the C++ side, or only a custom constructor for Python?

Reply:
The C++ Sample is an aggregate class so it has a default aggregate initializer which obviates the need for an explicit constructor to be defined: https://en.cppreference.com/w/cpp/language/aggregate_initialization

Python doesn't differentiate between aggregate initialization and constructors so you can export the initializer to Python as an ordinary constructor.